static/src/javascripts/projects/commercial/modules/dfp/prepare-googletag.js

```diff
@@ -125,6 +102,18 @@ export const init = (): Promise<void> => {
             }
         );
 
+        onConsentNotification('advertisement', state => {
+            if (state === true) {
```

Comment:
could simplify as below:

```
if (!_.isBoolean(state)) { return }
window.googletag.cmd.push(() => {
                    window.googletag.pubads().setRequestNonPersonalizedAds(state ? 1 : 0);
                });
```

Reply:
Great suggestion but because isBoolean is not used anywhere else in frontend this would mean a file size increase in the bundled js, albeit a small one. Coupled with the fact that this logic will most likely change in a very near future we decided to leave it for now and revisit when we go over it again.

Reply:
Great suggestion. `state` is boolean or null so we decided to use a slightly different version of your suggestions to avoid the extra import.